src/js/newsMenu.js

```diff
@@ -7,14 +7,8 @@ export class NewsBar{
     }
 
     async createNewsBar(publisher){
-        let data;
-
-        try{
-            data = await proxyApiFactory('get', newsApiQueryCreator.getNewsRecordsByPublisherNameQuery(publisher))
+        const data = await proxyApiFactory('get', newsApiQueryCreator.getNewsRecordsByPublisherNameQuery(publisher))
```

Comment:
please do not combine await and then, it's a bad approach
try something like:
const data = await proxyApiFactory('get', newsApiQueryCreator.getNewsRecordsByPublisherNameQuery(publisher));
const articles = data.articles.slice(0, 12);